app/experimenter/experiments/tests/api/v3/test_serializers.py

```diff
@@ -8,20 +9,39 @@
     ExperimentRapidSerializer,
     ExperimentRapidStatusSerializer,
 )
-from experimenter.experiments.models import Experiment, ExperimentChangeLog
+from experimenter.experiments.models import (
+    Experiment,
+    ExperimentChangeLog,
+)
 from experimenter.experiments.tests.factories import ExperimentFactory
 from experimenter.openidc.tests.factories import UserFactory
 from experimenter.base.tests.mixins import MockRequestMixin
 from experimenter.bugzilla.tests.mixins import MockBugzillaTasksMixin
 
 NIMBUS_DATA = get_data()
 FIREFOX_VERSION = random.choice(Experiment.VERSION_CHOICES)[0]
+FAKE_VARIANTS = [
+    {
+        "slug": "control",
+        "name": "control",
+        "ratio": 50,
+        "description": "a variant",
+        "is_control": True,
+    },
+    {
+        "slug": "variant",
+        "name": "variant",
+        "ratio": 50,
+        "description": "a variant",
+        "is_control": False,
+    },
+]
```

Comment:
I set a bad precedent here with putting these at the top of the file, we should try to keep test data that's only relevant to a single testcase inside the class, but we can deal with that later.

Reply:
I can add to the class instead if that's preferable